src/main/java/com/mercadolibre/dambetan01/model/Product.java

```diff
@@ -32,6 +32,4 @@
     @OneToOne
     private Stock stock;
 
-
-
 }
```

Comment:
Deixar relação de Produto com PurchaseOrder como ManoToMany

Reply:
Alterei pra many to many